pkgs/development/tools/build-managers/alire/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, gprbuild, gnat }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "alire";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "alire-project";
+    repo = "alire";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rwNiSXOIIQR1I8wwp1ROVOfEChT6SCa5c6XnTRqekDc=";
+
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ gprbuild gnat ];
+
+  # on HEAD (roughly 2c4e5a3), alire provides a dev/build.sh script. for now,
+  # just use gprbuild.
+  buildPhase = ''
+    gprbuild -j0 -P alr_env
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp -r ./bin $out
+  '';
+
+  meta = with lib; {
+    description = "A catalog of ready-to-use Ada/SPARK libraries plus a command-line tool (alr) to obtain, build, and incorporate them into your own projects";
+    homepage = "https://alire.ada.dev";
+    license = licenses.gpl3;
+    mainteners = with maintainers; [ atalii ];
```

```suggestion
    maintainers = with maintainers; [ atalii ];
```
ofborg caught a typo